x-pack/plugin/rollup/src/main/java/org/elasticsearch/xpack/rollup/v2/RollupShardIndexer.java

```diff
@@ -180,7 +182,13 @@ public void afterBulk(long executionId, BulkRequest request, BulkResponse respon
                             )
                         );
                     numFailed.addAndGet(failures.size());
-                    logger.error("Shard [{}] failed to populate rollup index. Failures: [{}]", indexShard.shardId(), failures);
+                    logger.info(
```

Comment:
If indexing rollup documents fail, we should fail the all the task. I think this is an error case and should be logged as error.

Reply:
Do you means that, if there are failed items, the total rollup task will be cancelled?

Reply:
Yes, otherwise we will end up with a wrong rollup index that does not contain all rollup documents. If we fail the whole task, ILM will handle the failure and retry

Reply:
got it, I will revert this change, and add the cancelled code when bulk failed